src/backend/libc/net/send_recv.rs

```diff
@@ -22,7 +23,10 @@ bitflags! {
         /// `MSG_DONTROUTE`
         const DONTROUTE = bitcast!(c::MSG_DONTROUTE);
         /// `MSG_DONTWAIT`
-        #[cfg(not(windows))]
+        #[cfg(not(any(
+            windows,
+            target_os = "aix",
+        )))]
         const DONTWAIT = bitcast!(c::MSG_DONTWAIT);
```

Comment:
On AIX, would it make sense to define `DONTWAIT` with the value of `MSG_NONBLOCK`?